sys-utils/umount.c

```diff
@@ -509,6 +510,15 @@ int main(int argc, char **argv)
 		case 'V':
 			print_version();
 			break;
+		case 'N':
+		{
+			int tmp;
+			if ((tmp = mnt_context_set_target_ns(cxt, optarg))) {
```

Comment:
What about to support in umount/umount (not in libmount)
```
   --namespace <pid|path>
```
where PID is interpreted as /proc/#pid/ns/mnt.

Reply:
OK